packages/orbit-components/src/Tooltip/index.js

```diff
@@ -16,6 +16,7 @@ const Tooltip = ({
   size = SIZE_OPTIONS.SMALL,
   content,
   preferredPosition,
+  lockScrolling,
```

Comment:
```suggestion
  lockScrolling = true,
```

Reply:
This seems unnecessary because it's ultimately being passed to `DialogContent`, which is the one that actually does something with it.

Reply:
but in the README file you have `default true`